test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -3657,7 +3696,10 @@ def matcher_check_fn():
         self._test_common(
             mod,
             (a,),
-            matcher_check_fn=matcher_check_fn,
+            matcher_check_fn,
+            include_ops=("qlinear_pointwise.binary_tensor",)
```

Comment:
Why don't we just check the `counters["inductor"]["qlinear_binary_matcher_count"]`?

Reply:
Now that I think of it. I should've used the counter.

For `[M=32, K=32, N=64]`, this pattern wasn't hit (and not even when I changed `K` to 48, it wasn't, so the counter was 0), and I thought this lowering wasn't because of those FX patterns. When I later discovered that only the `M=1` case was being lowered to `qlinear_binary`, I should've re-added the counter check.